README.md

```diff
@@ -35,183 +27,56 @@ OpenTTD is licensed under the GNU General Public License version 2.0,
-   this savegame to be able to reproduce the bug when the last savegame is not
-   old enough. If you loaded a scenario or savegame you need to attach that.
-- optionally you can attach the savegames from around 50%, 75%, 85%, 90% and
-   95% of the game's progression. We can use these savegames to speed up the
-   reproduction of the desync, but we should be able to reproduce these
-   savegames based on the first savegame and commands-out.log.
-- in case you use any NewGRFs you should attach the ones you used unless
-   we can easily find them ourselves via bananas or when they are in the
-   #openttdcoop pack.
-
-Do NOT remove the dmp_cmds savegames of a desync you have reported until the
-desync has been fixed; if you, by accident, send us the wrong savegames we
-will not be able to reproduce the desync and thus will be unable to fix it.
-
-## 3.0) Supported platforms
-
-OpenTTD has been ported to several platforms and operating systems. It should
-not be very difficult to port it to a new platform. The currently working
-platforms are:
+## 1.1) Downloading OpenTTD
+
+OpenTTD can be downloaded from the [official OpenTTD website](https://www.openttd.org/).
+
+Both 'stable' and 'nightly' versions are available for download:
+
+- most people should choose the 'stable' version, as this has been more extensively tested
+- the 'nightly' version includes the latest code changes and features, but may sometimes be less reliable
+
+On some platforms OpenTTD will also be available via your OS package manager or a similar service.
+
+
+## 1.2) OpenTTD gameplay manual
+
+OpenTTD has a [community-maintained wiki](https://wiki.openttd.org/), including a gameplay manual and tips.
+
+
+## 1.3) Supported platforms
+
+OpenTTD has been ported to several platforms and operating systems.
+
+The currently working platforms are:
 
 - FreeBSD (SDL)
 - Haiku (SDL)
-- Linux (SDL or Allegro)
+- Linux (SDL)
 - macOS (universal) (Cocoa video and sound drivers)
 - OpenBSD (SDL)
 - OS/2 (SDL)
-- Windows (Win32 GDI (faster) or SDL or Allegro)
-
-## 4.0) Installing and running OpenTTD
-
-Installing OpenTTD is fairly straightforward. Either you have downloaded an
-archive which you have to extract to a directory where you want OpenTTD to
-be installed, or you have downloaded an installer, which will automatically
-extract OpenTTD in the given directory.
+- Windows (Win32 GDI (faster) or SDL)
 
-OpenTTD looks in multiple locations to find the required data files (described
-in section 4.2). Installing any 3rd party files into a 'shared' location has
-the advantage that you only need to do this step once, rather than copying the
-data files into all OpenTTD versions you have.
+### 1.3.1) Legacy support
+Platforms, languages and compilers change.
+We'll keep support going on old platforms as long as someone is interested in supporting them, except where it means the project can't move the project forward to keep up with language and compiler features.
```

Comment:
Nitpick: 'the project can't move the project'?

Reply:
Good spot, now resolved thanks